### DISCUSSION TITLE
Kanban View [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2067] [Discord Threads]
> 🙋‍♂️ **MatthewBlott**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### Some contents in main website are outdated and we're revamping the website. the outdated content will be removed in the next release. Sorry for any inconvenience caused.  
#  
> 🙋‍♂️ **MatthewBlott**  
#### That answer sounds disingenuous. It's not 'some outdated content' but a premier feature clearly advertised the home page. It appears to have been left in place to entice users.  
#  
> 💁 **o1lab**  
#### Matthew, sorry you feel that way. But there is no premium feature or idea to 'entice' the users here. I believe you wouldnt the say same if you know the effort put by a small team to make this product free & open source (probably there is 0.5 million lines of code we are talking about). And there is a tonne of pro/premium features you may find in airtable are already free in nocodb! And we're making these feature work on any of your own existing database too! The snapshots you are talking about are proof of concept we 'd during the launch and it has taken more time/effort to accomplish existing features. If you feel strongly - you are welcome to send PRs or sponsor our effort.  
#  
> 🙋‍♂️ **MatthewBlott**  
#### I apologise if I got it wrong. I do appreciate the effort and I did make that point, I know FOSS teams are stretched. I still think it's a pretty bad oversight but I appreciate the reply.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972576417516765295)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
